src/scss/main.scss

```diff
@@ -1,6 +1,7 @@
 @import './settings';
 @import './spark-theme';
-@import '~@sparkdesignsystem/spark/spark.scss';
+@import '../../styles/spark';
```

Comment:
Remove this import. We only need the one from the styles package.

Reply:
I'll take care of this!